app/src/main/java/net/gsantner/markor/format/TextFormat.java

```diff
@@ -61,6 +66,7 @@ public class TextFormat {
             CONVERTER_ZIMWIKI,
             CONVERTER_KEYVALUE,
             CONVERTER_PLAINTEXT,
+            CONVERTER_EMBEDBINARY,
     };
 
     public static boolean isTextFile(final String absolutePath) {
```

Comment:
Strictily speaking, no of course it's not a textfile. But there won't be any other such case, either it's a binary file and is embedded in webview via file:// reference, or it is plaintext.